python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -164,7 +168,7 @@ def finishedIds(self):
         jobsIds : `set` [`int`]
             Set of integer job IDs.
         """
-        return set(job.qdata.index for job in self.jobs if job.state == JobState.FINISHED)
+        return set(job.qdata for job in self.jobs if job.state == JobState.FINISHED)
```

Comment:
Should it be `job.index` instead? Same for two other changes below. If it needs to return nodes instead of index then it's better to rename methods and update docstrings.